src/tools/gophish_export.py

```diff
@@ -253,6 +254,74 @@ def get_application(rawEvent):
     return application
 
 
+def find_unique_target_clicks_count(clicks):
+    """Return the number of unique clicks in a click dict."""
+    uniq_list = list()
+    for click in clicks:
+        if click["user"] not in uniq_list:
+            uniq_list.append(click["user"])
+    return len(uniq_list)
+
+
+def write_assessment_click_summary(api, assessment_id):
+    """Output a click summary report to JSON, console, and a text file."""
+    assessment_click_summary = dict()
+    campaign_click_stats = dict()
+    campaigns = list()
+    campaign_ids = get_campaign_ids(api, assessment_id)
+    num_campaigns = len(campaign_ids)
+    summary_json = assessment_id + "_click_summary.json"
+    summary_outfile_name = assessment_id + "_click_summary.txt"
+    summary_outfile = open(summary_outfile_name, "w")
```

Comment:
Perhaps something like this to improve the readability of the variable names?  Obviously you'd also need to update these variable names throughout this function if you choose to accept this suggestion.

```suggestion
    json_filename = assessment_id + "_click_summary.json"
    text_filename = assessment_id + "_click_summary.txt"
    summary_outfile_text = open(text_filename, "w")
```

Reply:
I still would like to see better variable names here.  As of https://github.com/cisagov/gophish-tools/pull/40/commits/0a3f4bc0587431f616a0d35548f9fc80da72a16b, you have them called `json_filename` and `summary_outfile_name`.  I'd rather see more similarly-named variables (e.g. `json_filename` / `text_filename` or `summary_json_filename` / `summary_text_filename`).

Reply:
These variable names were cleaned up in https://github.com/cisagov/gophish-tools/pull/40/commits/0a0c1649c0f6a2b3debf21bc8f1682c4eb37a8d5 - thanks!